src/Analyzers/Analyzers/Serializability/ViewModelSerializabilityAnalyzer.cs

```diff
@@ -222,6 +223,13 @@ private static void AnalyzePropertyType(ITypeSymbol propertyType, Location locat
                     }
                     break;
 
+                case TypeKind.TypeParameter:
+                    // Assume this is correct. This gets otherwise complicated. Possible solution:
+                    // 1.) Check for available generic type constraints (these might hint whether the type is serializable or not)
+                    // 2.) Check the context this viewmodel is used in (i.e. analyze only concrete instantiations of the generic type definition)
+                    // TODO: Maybe add (partial) support for this in future
+                    break;
```

Comment:
yea, I doubt it's so commonly used and this is only a hint to users, the validation does not need to be bullet proof